Dockerfile_devel

```diff
@@ -7,20 +7,20 @@ RUN apt-get update &&\
         net-tools \
```

Comment:
This file is so far from maintained in fmriprep. Do you use it? I would be inclined to drop it.

Reply:
No I've never used the devel Dockerfiles. I'll remove it.